src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+import { get } from 'http';
 
-export class Triangle implements Figure {}
+interface Figure {
+  getArea(): number;
+  getInfo(): string;
+}
+
+class Triangle implements Figure {
+  constructor(
+    private color: string,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side length cannot be zero or negative');
```

Comment:
[CODE STYLE] - When throwing an error, it's best to provide a more detailed description in the error message to make it clear what the problem is. Here, you could specify that side lengths of a triangle are being checked.